CSS/footer.css

```diff
@@ -0,0 +1,216 @@
+}
+.footer__navigation-list {
+    margin-top: 20px;
+
+}
+.navigation-list {
+    display: flex;
+    flex-direction: column;
+    justify-content:space-between;
+    font-style: normal;
+    letter-spacing: normal;
+    line-height: normal;
+}
+.list__item {
+    list-style-type: none; 
+    flex: 1 1 auto; 
+    align-items: baseline; 
+    margin-bottom: 13px;         
+}
+.list__link {
+    text-decoration: none;
+    color: #ffffff;
+    font-family: Roboto;
+    font-size: 14px;
+    font-weight: 400;
+    font-style: normal;
+    letter-spacing: normal;
+    line-height: normal;
+    text-align: left;
+}
+.footer__phone-number {
+    margin: 0px 0px 27px 0px;
+}
+.footer__icons {
+    font-size: 23px;
+    width: flex;
+    letter-spacing: 6px;
+    margin-bottom: 28px;
+}
+.footer__item-line {
+    height: 1px;
+    background-color: rgba(255, 255, 255, 0.452);
+    width: 100%;
+    margin: 0px;
+    border: none;
+}
+.footer__copyright {
+    font-family: 'Roboto', sans-serif; 
+    font-size: 0.7em;
+    letter-spacing: 0.1em;
+    color: rgba(255, 255, 255, 0.452);
+    word-spacing: 2px;
+    margin: 10px 0px;
+}
+
+
+@media only screen and (min-width: 768px) {
+
+.footer__body {
+    display: flex;
+    flex-direction: row;
+    
+    flex: 1 1 auto;
+    margin: 0px 0px 0px 0px;
+}
+.footer__logo-container {
+    order: 1;
+    margin: 32px 50px 34px 20px;
+}
+.logo {
+    color: white;
+    font-family: none;
+    font-size: 20px;
+    font-weight: 500;
+    text-align: left;
+    letter-spacing: 0.6px;
+}
+.footer__navigation-list {
+    margin: auto;
+    width: 84%;
+    order: 2;
+}
+.navigation-list {
+    flex-direction: row;
+    width: 100%; 
+}
+.list__item {
+    text-align: left;
+    margin: 0px 0px 0px 8px;
+}
+.list__link {
+    font-size: 12px;
+    font-weight: 400;
+    color: #ffffff;
+}
+.footer__phone-number {
+    display: none;
+}
+.footer__item-line {
+    order: 3;
+    margin: 0px 66px 14px 66px;
+}
+.footer__copyright { 
+    order: 4;
+    flex-basis: 47%;
+    flex-grow: 0;
+    align-self: center;
+    margin: auto;
+    margin-bottom: 19px;
+    text-align: right;
+}
+.footer__icons {
+    order: 5;
+    flex-basis: 45%;
+    flex-grow: 0;
+    align-self: center;
+    margin: auto;
+    margin-bottom: 13px;
+}
+
+}
+
+@media only screen and (min-width: 1366px) {
+
+.footer {
+    background-color: #e8e8e8;
+} 
+.footer__body {
+    margin: 0px 73px;
+}  
+.footer__logo-container {
+    order: 1;
+    margin: 30px 0px 37px 0px;
+}
+.logo {
+    color: black;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 43.87px;
+    letter-spacing: 0.4px;
+}
+.footer__navigation-list {
+    width: 64%;
+    margin: 43px 0px auto 162px;
+    order: 2;
+}
+.navigation-list {
+    flex-direction: row;
+}
+.list__item {
+   margin: 0px;
+}
+.list__link {
+    font-family: 'circe', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    color: #000000;
+} 
```

Comment:
This elements should be part of `navigation-list` bem class. Also `list` bem class is too generic and it will make other people think that they can reuse it for evry list that they have. Please, make it a part of `navigation-list`.

Also it is a general practice to disable margin on the last list item. You shold look for `:last-of-type` pseudo class.